bindata/assets/kueue-operator/deployment.yaml

```diff
@@ -82,8 +80,6 @@ spec:
       serviceAccountName: kueue-controller-manager
       terminationGracePeriodSeconds: 10
       volumes:
```

Comment:
When I enabled Read Only filesystem for Kueue, I now remember that there were some things I had to fix for visibility.

```
{"level":"error","ts":"2025-09-26T19:54:26.762519999Z","logger":"setup","caller":"kueue/main.go:268","msg":"Unable to create and start visibility server","error":"unable to apply VisibilityServerOptions: error creating self-signed certificates: open /visibility/apiserver.crt: read-only file system","stacktrace":"main.main.func3\n\t/workspace/upstream/kueue/src/cmd/kueue/main.go:268"}
```

https://github.com/kubernetes-sigs/kueue/pull/5059/files

Reply:
@kannon92 where you saw that error? I can't find it in the logs

Reply:
In the must gather. 

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_kueue-operator/728/pull-ci-openshift-kueue-operator-main-test-e2e-4-18/1971660190885351424/artifacts/test-e2e-4-18/kueue-must-gather/artifacts/must-gather/quay-io-redhat-user-workloads-kueue-operator-tenant-kueue-must-gather-main-sha256-0ab563ec73136b1e973ba26e448250289cc6a2a5020019ed4f1662bbea3db055/kueue/logs-controller-manager-kueue-controller-manager-7b548447c8-d7fzw.log

I always check the logs for Kueue/operator.